### ISSUE TITLE
the 0.14.1 branch release still has  the Animation issue (but the 0.14.1 tag hasn't)

### ISSUE THREAD

**Issue by xx1adfasd** [bevyengine/bevy#14610]
Related to 
https://github.com/bevyengine/bevy/pull/14411/files#diff-2b9739c060b50101f9ab4e0bab00c34bcc80201a52ab13315b08a649765eb45f

https://github.com/bevyengine/bevy/issues/13910

and my issue
https://github.com/bevyengine/bevy/issues/14548

Why is it still not fixed in 0.14.1?

I still see 

```
 self.main_animation = Some(new_animation);
player.start(new_animation)
```

instead of 

```
self.transitions
    .retain(|transition| transition.animation != new_animation);

player.start(new_animation)
```

I can see it in the source code
https://github.com/bevyengine/bevy/blob/7c80ae73133f9866f2ed477ce9922c83fc3fe1cc/crates/bevy_animation/src/transition.rs#L97

but, when I use it in my code, in the fetched file

/Users/my_user_name/.cargo/registry/src/index.crates.io-6f17d22bba15001f/bevy_animation-0.14.1/src/transition.rs

I can still see the old code instead of new.

Why?? I'm confused.

Can somebody confirm this? The fetched code from crates.io isn't the same as the github code.

**Comment by xx1adfasd** [comment]
Ok, I can see that in the branch release-0.14.1

https://github.com/bevyengine/bevy/blob/release-0.14.1/crates/bevy_animation/src/transition.rs

the bug is not fixed,

but in the git tag v0.14.1, 

https://github.com/bevyengine/bevy/blob/v0.14.1/crates/bevy_animation/src/transition.rs

the bug is fixed.

So, It's not my problem. There's something wrong with the git tag and the git branch

**Comment by xx1adfasd** [comment]
Another question.

Why in the tag 0.14.1, in the Cargo.toml, it says `version = "0.15.0-dev"`?

I think if you tag as 0.14.1, the version should go as 0.14.1 as well.

**Comment by mockersf** [comment]
Thanks for noticing!

It looks like I missed this PR when cherry picking for the patch. I added it to the 0.14.2 milestone, hopefully I won't miss it a second time. Sorry about that.

I also fixed the v0.14.1 tag
